composer.json

```diff
@@ -31,5 +32,10 @@
     },
     "config": {
         "bin-dir": "bin/"
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "tantrum_elastic\\Lib\\Build::initialiseContainer"
+        ]
```

Comment:
This will build the ```container.ser``` bin file, so I think it should be excluded from the repo. Otherwise would this file not be in an updated state after every ```composer install```?